src/plot_api/plot_api.js

```diff
@@ -195,8 +198,10 @@ Plotly.plot = function(gd, data, layout, config) {
     function marginPushersAgain() {
         // in case the margins changed, draw margin pushers again
         var seq = JSON.stringify(fullLayout._size) === oldmargins ?
-            [] : [marginPushers, subroutines.layoutStyles];
-        return Lib.syncOrAsync(seq.concat(Fx.init), gd);
+            [] :
+            [marginPushers, subroutines.layoutStyles];
+
+        return Lib.syncOrAsync(seq, gd);
```

Comment:
no need to call `Fx.init` here too as it is called in `drawFramework` on every `Plotly.plot`.

Previously, we need a change of subplot in order to trigger `Fx.init`.